__tests__/app.test.js

```diff
@@ -37,3 +40,17 @@ describe("GET /api/topics", () => {
     });
   });
 });
+
+describe("GET /api/", () => {
+  test("get all APIs with the decription, queries and example", async () => {
+    const res = await request(app).get("/api").expect(200);
+    const endpoints = res.body;
+    for (const key in endpoints) {
+      expect(endpoints[key]).toMatchObject({
```

Comment:
by requiring in the actual endpoints.json file, we can compare the result to that and see if it matches. It will then be a stricter test